src/libs/DebugUtils.ts

```diff
@@ -480,6 +480,7 @@ function validateReportDraftProperty(key: keyof Report | keyof ReportNameValuePa
         case 'isWaitingOnBankAccount':
         case 'isCancelledIOU':
             return validateBoolean(value);
+        case 'exportFailedTime':
```

Comment:
`exportFailedTime` is a numeric value:

<img width="342" alt="Screenshot 2025-04-15 at 12 27 58 p m" src="https://github.com/user-attachments/assets/b20fc01e-9a10-4198-b81c-df48309f1db4" />